rest-openehr/src/main/java/org/ehrbase/rest/openehr/controller/OpenehrCompositionController.java

```diff
@@ -315,6 +322,9 @@ public ResponseEntity deleteComposition(@ApiParam(value = REQ_OPENEHR_VERSION) @
      * "{?version_at_time}" is hidden in swagger-ui, it only is here to be piped through.
      */
     @GetMapping("/{ehr_id}/composition/{version_uid}")
+    // checkAbacPre /-Post attributes (type, auth object, subject, payload, content type)
+    @PostAuthorize("checkAbacPost(@openehrCompositionController.COMPOSITION, authentication, "
+        + "@ehrService.getSubjectExtRef(#ehrIdString), returnObject, #accept)")
```

Comment:
I will test now because I am not 100% sure may be the `returnObject` is not needed here